src/nsm.h

```diff
@@ -26,6 +26,8 @@
 /*                                                           */
 /* #include "nsm.h"                                          */
 /*                                                           */
+/* static nsm_client_t *nsm = 0                              */
```

Comment:
value should be NULL, also missing ; at the end

Reply:
actually, the code here always uses 0 instead of NULL, so ignore that. but still missing ;

Reply:
Yes, you are right. This is pseudo code, and this line is taken directly from the original, means original use 0 and miss the semicolon.

Reply:
Fixed now.